src/backend/egl/display.rs

```diff
@@ -490,7 +683,7 @@ impl EGLBufferReader {
     pub fn egl_buffer_contents(
```

Comment:
Looks like this doc is outdated (the original buffer is not returned, given it is only passed by reference).